environments/production/postgresql.yml

```diff
@@ -57,25 +57,43 @@ dbs:
   form_processing:
     proxy:
       host: pgbouncer3
-      pgbouncer_host: pgbouncer3
+      pgbouncer_endpoint: pgbouncer3
+      pgbouncer_hosts:
+        - pgbouncer3
+        - pgbouncer9
     partitions:
       p1:
         shards: [0, 204]
         host: rds_pgshard1
-        pgbouncer_host: pgbouncer3
+        pgbouncer_endpoint: pgbouncer9
+        pgbouncer_hosts:
+          - pgbouncer3
```

Comment:
Was pgbouncer3 included here so we can revert back to the previous configuration in case this goes wrong, or was there another reason?

Reply:
I assume that's the reason yes.